cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java

```diff
@@ -340,6 +341,9 @@
     private final AeshCommands aeshCommands;
     private CLICommandInvocation invocationContext;
     private final CommandCompleter legacyCmdCompleter;
+
+    private ColorConfig colorConfig;
```

Comment:
This field is not used, the colorConfig is retrieved directly from the config.

Reply:
It is used on line 435